docs/optimization-framework/ab-testing.md

```diff
@@ -85,30 +87,141 @@ Returns the A/B Test markup for client side processing.
 - `$default_content`: The default content for the control test.
 - `$args`: An optional array of data to pass through to `variant_callback`.
 
+### Example basic test
+
+An example for testing a CTA (call to action) button with different text would look like:
```

Comment:
It's hard from the code example to see how this would be written or used from start to finish. I think it needs to be something easier to grok and more real-world so perhaps we can reuse the bulk of the code for the title test but apply it to a piece of post meta data, then follow that through showing the post meta being filtered and used in a template part on output.